clients/rospy/src/rospy/impl/tcpros_base.py

```diff
@@ -157,7 +157,7 @@ def run(self):
             except ConnectionAbortedError:
                 continue
             except IOError as e:
-                (e_errno, msg) = e.args
+                (e_errno, msg, *discard) = e.args
```

Comment:
Please use `_` instead of `discard`.

Reply:
You mean using `*_` right? I renamed it in the new commit.